admin/spec/components/solidus_admin/base_component_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe SolidusAdmin::BaseComponent, type: :component do
+  describe "#spree" do
+    it "gives access to spree routing helpers" do
+      # The spree/core routes start as empty, so we need to add a route to test
```

Comment:
Didn't we add the products resource routes in the previous commit?

Reply:
For `SolidusAdmin` we have routes that are isolated from `Spree::Core`, and the previous commit only added the resource to the `SolidusAdmin::Engine` routes, not the `Spree::Core::Engine` ones.

Reply:
Ah, got it! Makes sense, thanks.